rites/roman1969/src/calendars/countries/hong-kong/index.ts

```diff
@@ -0,0 +1,81 @@
+import { PatronTitle } from '../../../constants/martyrology-metadata';
+import { Precedences } from '../../../constants/precedences';
+import { CalendarDef } from '../../../models/calendar-def';
+import { Inputs } from '../../../types/calendar-def';
+
+export class HongKong extends CalendarDef {
+  particularConfig = {
+    ascensionOnSunday: true,
+    epiphanyOnSunday: true,
+    corpusChristiOnSunday: true,
+  };
+
+  inputs: Inputs = {
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    mary_mother_of_god: {
+      isHolyDayOfObligation: false,
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    joseph_freinademetz_priest: {
+      precedence: Precedences.ProperMemorial_11b,
+      dateDef: { month: 1, date: 29 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    gabriele_maria_allegra_priest: {
+      precedence: Precedences.ProperMemorial_11b,
+      dateDef: { month: 1, date: 30 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    aloysius_versiglia_bishop_and_callistus_caravario_priest_martyrs: {
+      precedence: Precedences.OptionalMemorial_12,
+      dateDef: { month: 2, date: 25 },
+      martyrology: ['aloysius_versiglia_bishop', 'callistus_caravario_priest'],
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    our_lady_of_china: {
+      precedence: Precedences.ProperMemorial_11b,
+      dateDef: { month: 5, dayOfWeek: 6, nthWeekInMonth: 2 },
+    },
+
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm
+    our_lady_help_of_christians: {
+      precedence: Precedences.OptionalMemorial_12,
+      dateDef: { month: 5, date: 24 },
+    },
```

Comment:
According to this source, the celebration is celebrated as a memorial.

```suggestion
    // src: http://catholic-dlc.org.hk/viewer.htm?doc=0524e.doc
    our_lady_help_of_christians: {
      precedence: Precedences.ProperMemorial_11b,
      dateDef: { month: 5, date: 24 },
    },
```